### DISCUSSION TITLE
Documentation needs working examples added

### DISCUSSION THREAD

**Discussion by d3vl1m3** [oblador/loki#413] [General]
I'm looking at the command line arguments and it's great these exist. But the documentation only tells us what the default argument is. 

Some _real-world_ examples of what their usage looks like would really help.

The ones I'm looking at right now are:
- `--skipStories` (don't know what `${kind} ${story}` are meant to represent.
- `--storiesFilter` 
- `--targetFilter `

I'd like to test specific screen sizes only so I can batch jobs in the pipeline and figure one of these are what I'm looking for I just have no idea how to format the arguments right now.

**Comment by peterblazejewicz** [comment]
@d3vl1m3 this looks like string pattern used to include or exclude (depending on the option) stories based on titles, e.g. `storiesFilter` is being assigned to `includePattern` and used as:
```js
return new RegExp(includePattern, 'i').test(fullStoryName);
```
